wget_check.sh

```diff
@@ -19,8 +19,15 @@ function wget_check {
         restore_e=1
         set +e
     fi
+
     # bash -c is used to avoid issues due to quotation within quotation
-    bash -c "wget ${WGET_OPT} -O $file_name $url"
+    if [[ $url =~ ^s3:\/\/ ]]; then
+        export PATH=$HOME/.local/bin:$PATH
+        bash -c "aws s3 cp --only-show-errors $url $file_name"
```

Comment:
Seems a bit misleading to leave this function as "wget_check" when it isn't using wget.